src/Adaptivity/AdaptiveCells.jl

```diff
@@ -46,7 +46,7 @@ end
 #OctantBWG(dim::Int,l::Int,m::Int,b::Int=_maxlevel[dim-1]) = OctantBWG(dim,l,m,b)
 #OctantBWG(dim::Int,l::Int,m::Int,b::Int32) = OctantBWG(dim,l,m,b)
 #OctantBWG(dim::Int,l::Int32,m::Int,b::Int32) = OctantBWG(dim,l,Int32(m),b)
-function OctantBWG(level::Int,coords::NTuple)
+function OctantBWG(level::T,coords::NTuple) where T <: Integer
```

Comment:
```suggestion
function OctantBWG(level::Integer,coords::NTuple)
```
Is there a reason why you grab T explicitly?

Reply:
I thought that we could add later a check if `level` and `eltype(coords)` matches and use the wider/smaller type